modules/store/migrations/migration.json

```
@@ -15,6 +15,11 @@
       "description": "The road to v8 RC",
       "version": "8-rc.1",
       "factory": "./8_0_0-rc/index"
+    },
+    "ngrx-store-migration-13-beta": {
```

Based on the previous migration names:

```suggestion
    "ngrx-store-migration-04": {
```

---

Yea... 😅
The name doesn't really matter (it's only important to run the schematic in the test).
I was thinking it would make more sense to follow the versions, instead of creating an own counter.